extensions/desktop-common/desktop-init

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -e
+
+#################
+# Launcher init #
+#################
+
+# On Fedora $SNAP is under /var and there is some magic to map it to /snap.
+# # We need to handle that case and reset $SNAP
+SNAP="${SNAP/\/var\/lib\/snapd/}"
```

Comment:
this is not exported nor used in this script, might need to be moved or exported

Reply:
This has been dropped for now in changeset fd9cdd9.